salt/modules/zypper.py

```diff
@@ -628,16 +628,16 @@ def install(name=None,
         log.info('Targeting repo {0!r}'.format(fromrepo))
     else:
         fromrepoopt = ''
+    cmd_install = ['zypper', '--no-refresh', '--non-interactive', 'install',
```

Comment:
@isbm 
`install` already has `refresh=False` argument. In case `refresh=True` it calls `zypper refresh` (`refresh_db` function). I don't see the point actually. I would use `refresh=False <-> --no-refresh` mapping.